wbooks/src/app/screens/booklist/index.js

```diff
@@ -1,15 +1,35 @@
 import React, { Component } from 'react';
-import { View, StyleSheet } from 'react-native';
+import { 
+	SafeAreaView, 
+	View, 
+	StyleSheet, 
+	FlatList 
+} from 'react-native';
 import BookItem from './components/BookItem';
 import styles from './styles';
+import books from './books';
 
 class BookList extends Component {
 
+	renderItem = ({ item }) => (
+		<BookItem 
+			title= {item.title}
+			writer= {item.author}
+			uri= {item.image_url}
```

Comment:
You should use camelCase

Reply:
Mh dont worry about this now, i think u will have to wait to see Redux to fix this, so dont worry just skip it.

Reply:
will be changed later